### DISCUSSION TITLE
Python SDK for SAF

### DISCUSSION THREAD

**Discussion by jarabroz** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#296] [General]
Dear @scia-nv ,

Are you planning to publish some Python SDK for work with SAF format? It would be nice to have an option for reading and writing SAF objects from/to MS Excel. Could be also nice some kind of data validation option

**Comment by jorgepz** [comment]
I join in! In [ONSAS](https://github.com/ONSAS) we are considering having SAF as an I/O format, thus having an SDK either in Python, Julia or C++ would be very useful.
